src/ckb-daemon/usb.h

```diff
@@ -159,7 +160,7 @@ const char* product_str(short product);
 #define IS_MONOCHROME_DEV(kb)           IS_MONOCHROME((kb)->vendor, (kb)->product)
 
 /// Full color range (16.8M) vs partial color range (512)
-#define IS_FULLRANGE(kb)                (IS_RGB((kb)->vendor, (kb)->product) && (kb)->product != P_K65 && (kb)->product != P_K70 && (kb)->product != P_K95)
+#define IS_FULLRANGE(kb)                (IS_RGB((kb)->vendor, (kb)->product) && (kb)->product != P_K65 && (kb)->product != P_K68 && (kb)->product != P_K70 && (kb)->product != P_K95)
```

Comment:
I'm pretty sure this is a mistake, Casey. This define is for full 24-bit colour, and I doubt Corsair would reintroduce 512 colour in the K68 RGB.